README.md

```diff
@@ -111,7 +111,8 @@ Options:
   - `basedir`: Set this if you need to require node modules in `node` mode
   - `static`: Serve static files from this directory at `/`
   - `loc`: a full url like `http://localhost:8080/` for using an existing http server. When `loc` is supplied, options `node`, `basedir`, and `static` are all ignored.
-
+  - `sandbox`: Run electron with sandbox (default); use "--no-sandbox" to emit debug information when using Docker
```

Comment:
```suggestion
  - `sandbox`: Run electron with sandbox. Disable to emit debug information when using Docker. Defaults to `true`.
```

This section is describing the `API`, not the `CLI`, so it would be `sandbox: false`, not `--no-sandbox`. We don't have `CLI` documentation for this yet, so no need to document `--no-sandbox`.